package br.com.cdb.bancodigital.seguranca.config.exception;

import org.springframework.http.HttpStatus;

import java.io.Serial;

public class ResourceAlreadyExistsException extends ApiException {

    @Serial
    private static final long serialVersionUID = 1L;

    public ResourceAlreadyExistsException(String message) {
        super(message, HttpStatus.CONFLICT);
    }
}
